src/main/java/uk/gov/ons/census/fwmtadapter/messaging/CaseUpdatedReceiver.java

```diff
@@ -53,4 +96,15 @@ private boolean canIgnoreEvent(ResponseManagementEvent event) {
     return event.getPayload().getMetadata() == null
         || event.getPayload().getMetadata().getFieldDecision() == null;
   }
+
+  private Boolean getCEComplete(CollectionCase caze) {
+    if (caze.getAddress().getAddressType().equals("CE")
+        && caze.getAddress().getAddressLevel().equals("E")) {
+      if (caze.getReceiptReceived() != null) {
```

Comment:
Could be anded instead of another nested `if`?

Reply:
done